src/Microsoft.Tye.Core/ProjectReader.cs

```diff
@@ -268,6 +268,10 @@ private static void EvaluateProject(OutputContext output, ProjectServiceBuilder
             project.Frameworks.AddRange(sharedFrameworks.Select(s => new Framework(s)));
             output.WriteDebugLine($"Found shared frameworks: {string.Join(", ", sharedFrameworks)}");
 
+            // determine container base image
+            project.ContainerInfo!.BaseImageName = projectInstance.GetPropertyValue("ContainerBaseImage");
```

Comment:
I think having a null check here for project.ContainerInfo is probably better.

Reply:
Need the property here, should I create it?
`new ContainerInfo() { UseMultiphaseDockerfile = false, };`